src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -10,8 +10,10 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Microsoft.CodeAnalysis.VisualBasic.SyntaxFacts
 Imports Roslyn.Test.Utilities
 Imports Xunit
+Imports Microsoft.CodeAnalysis.Collections
 
 Friend Module ParserTestUtilities
+    Friend ReadOnly Property _PooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = PooledStringBuilder.CreatePool()
```

Comment:
Changing this one to `PooledStringBuilderPool` as it still needs to be `Friend`, as it called in helper module further down.